_content/essays/real-sal-flower_dhammika.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "What is the Real Sal Flower?"
+authors:
+  - dhammika
+external_url: "https://www.lotuslibrary.com/ebooks/what-is-the-real-sal-flower/"
```

Comment:
Please you Bhante's own site: https://budblooms.org/2020/05/10/what-is-the-real-sal-flower/